src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -270,8 +235,42 @@ public abstract partial class JsonNode
         public abstract System.Text.Json.JsonNode Clone();
         public static System.Text.Json.JsonNode DeepCopy(System.Text.Json.JsonElement jsonElement) { throw null; }
         public static System.Text.Json.JsonNode GetNode(System.Text.Json.JsonElement jsonElement) { throw null; }
-        public static System.Text.Json.JsonNode Parse(string json) { throw null; }
+        public static implicit operator System.Text.Json.JsonNode (bool value) { throw null; }
+        public static implicit operator System.Text.Json.JsonNode (byte value) { throw null; }
+        public static implicit operator System.Text.Json.JsonNode (System.DateTime value) { throw null; }
+        public static implicit operator System.Text.Json.JsonNode (System.DateTimeOffset value) { throw null; }
+        public static implicit operator System.Text.Json.JsonNode (decimal value) { throw null; }
+        public static implicit operator System.Text.Json.JsonNode (double value) { throw null; }
+        public static implicit operator System.Text.Json.JsonNode (System.Guid value) { throw null; }
+        public static implicit operator System.Text.Json.JsonNode (short value) { throw null; }
+        public static implicit operator System.Text.Json.JsonNode (int value) { throw null; }
+        public static implicit operator System.Text.Json.JsonNode (long value) { throw null; }
+        [System.CLSCompliantAttribute(false)]
+        public static implicit operator System.Text.Json.JsonNode (sbyte value) { throw null; }
+        public static implicit operator System.Text.Json.JsonNode (float value) { throw null; }
+        public static implicit operator System.Text.Json.JsonNode (string value) { throw null; }
+        [System.CLSCompliantAttribute(false)]
+        public static implicit operator System.Text.Json.JsonNode (ushort value) { throw null; }
+        [System.CLSCompliantAttribute(false)]
+        public static implicit operator System.Text.Json.JsonNode (uint value) { throw null; }
+        [System.CLSCompliantAttribute(false)]
+        public static implicit operator System.Text.Json.JsonNode (ulong value) { throw null; }
+        public static System.Text.Json.JsonNode Parse(string json, System.Text.Json.JsonReaderOptions options = default(System.Text.Json.JsonReaderOptions), System.Text.Json.DuplicatePropertyNameHandling duplicatePropertyNameHandling = System.Text.Json.DuplicatePropertyNameHandling.Replace) { throw null; }
```

Comment:
Ok, for now I changed it `JsonReaderOptions` into `JsonDocumentOptions`. I am not sure if we should add `DuplicatePropertyNameHandling` to `JsonDocumentOptions`, because `JsonDocument` does not support it, so users will be confused by seeing that setting this option does not change anything it this case.

Reply:
I think it's fine for `JsonDocumentOptions` to throw something like `ArgumentException` while we don't have support for `DuplicatePropertyNameHandling`. Having a separate option being passed-in doesn't scale well. If we want to add a new option that is specific to JsonNode/JsonDocument, would we then add a new Parse overload?

Also, we can re-visit this if JsonDocument cannot support the various `DuplicatePropertyNameHandling` modes, and maybe create a JsonNodeOptions struct that is similar to JsonReaderOptions/JsonDocumentOptions, but with the extra option for duplicates.

Reply:
Yes, probably `JsonNodeOptions` would be more appropriate.